src/nvim/event/libuv_process.c

```diff
@@ -41,7 +41,7 @@ int libuv_process_spawn(LibuvProcess *uvproc)
 #endif
   uvproc->uvopts.exit_cb = exit_cb;
   uvproc->uvopts.cwd = proc->cwd;
-  uvproc->uvopts.env = NULL;  // Inherits the parent (nvim) env.
+  uvproc->uvopts.env = proc->env;
```

Comment:
should keep the comment explaining what NULL does, or maybe add it to the docstring for `channel_job_start()`.

Reply:
> should keep the comment explaining what NULL does, or maybe add it to the docstring for `channel_job_start()`.

I added a docstring for `channel_job_start()`, including this tidbit.